src/connection/server_support.c

```diff
@@ -1816,24 +1821,27 @@ css_pack_message_to_master (const char *server_name)
     */
 
   sprintf (pid_string, "%d", getpid ());
+
+  /* | server_type (1 byte) | '#' if HA-mode enabled (1 byte) | server_name | release string | env_name | pid | */
```

Comment:
The comment should match the order of definition of `message_size`.
```suggestion
  /* | server_type (1 byte) | server_name | '#' if HA-mode enabled (1 byte) | release string | env_name | pid | */
```

Reply:
Or the order in the definition of `message_size` according to the order of those are appended.

Reply:
I will make sure to align 
 the actual order in which the components are appended, 
 the order in which the component sizes are added in the `message_size`, 
and the order described in the comments.

Reply:
GOOD

Reply:
Done, by adding `ha_token_size`